### DISCUSSION TITLE
possible to have the wireless controller join home wifi?

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#238] [Ideas]
i have many esp32 projects already, would be cool/ useful to have it attempt to join home wifi then fail back to broadcasting its own network,

could we also change the hostname? instead of proton pack xxxx it could be user configurable?

**Comment by DustinGrau** [comment]
This feature has been added for some time (external wifi support as a client), just didn't realize there was a request in here for the same. For now, the name of the WiFi will not be made user-editable.
